Sources/xcodeproj/Objects/Project/PBXOutputSettings.swift

```diff
@@ -120,6 +120,17 @@ public enum PBXBuildPhaseFileOrder {
     }
 }
 
+/// Defines the format of project file references
+public enum PBXReferenceFormat {
```

Comment:
Nitpick: Is this the right format for documenting enums? IIRC the documentation of the cases goes in the enum documentation.

Reply:
This is consistent with the documentation of other enums in this class.

Reply:
I checked, and while the automatic doc shortcut does generate a single doc with bullet points for each of the case statements, the docs are not carried onto the case statements, meaning they effectively appear undoc'd if inspecting them. The way we have it here allows you to inspect them individually. My opinion is this format is better for consumers.

Reply:
Fair enough. Being able to inspect them individually is a great argument to keep the format that you proposed.